demo/ocs/config/run-ocs

```diff
@@ -2,14 +2,14 @@
 #
 # Copyright 2020 Intel Corporation
 # SPDX-License-Identifier: Apache 2.0
-#
-# Specify the proxy here. For example, if the http proxy is 'http://myproxy.com:900',
-# then http_proxy_host would be 'myproxy.com' and http_proxy_port would be '900'.
-# If no proxies are needed, leave the fields empty.
-http_proxy_host=
-http_proxy_port=
-https_proxy_host=
-https_proxy_port=
+
+SYSTEM_PROPERTIES="-Dserver.port=${SERVER_PORT} -Dlogging.config=${LOGGING_CONFIG} -Dapplication.version=${APPLICATION_VERSION}"
```

Comment:
We can change these statements to following format to improve readability. Same for FILE_SYSTEM_PROPERTIES, TO_PROPERTIES and JAVA_SPRING_TLS_PROPERTIES.

SYSTEM_PROPERTIES="-Dserver.port=${SERVER_PORT}"  
SYSTEM_PROPERTIES+=" -Dlogging.config=${LOGGING_CONFIG}" # A space is required between '"' and '-' at the beginning would ensure separation of flags
SYSTEM_PROPERTIES+=" -Dapplication.version=${APPLICATION_VERSION}"

Reply:
Done. To make this way of string concatenation work, this script will be run inside the container using 'bash', instead of 'sh'.

Reply:
Done. To make this way of string concatenation work, this script will be run inside the container using 'bash', instead of 'sh'.